dev/com.ibm.ws.jaxrs.2.0_fat/test-applications/callback/src/com/ibm/ws/jaxrs/fat/callback/Resource.java

```diff
@@ -0,0 +1,193 @@
+package com.ibm.ws.jaxrs.fat.callback;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import javax.ws.rs.core.Response.Status;
+
+@Path("resource")
+public class Resource {
+
+    public static final String RESUMED = "Response resumed";
+    public static final String ISE = "Illegal State Exception Thrown";
+    public static final String NOE = "No Exception Thrown";
+    public static final String FALSE = "A method returned false";
+    public static final String TRUE = "A method return true";
+
+    //
+    private static final AsyncResponseBlockingQueue[] stage = {
+                                                               new AsyncResponseBlockingQueue(1),
+                                                               new AsyncResponseBlockingQueue(1),
+                                                               new AsyncResponseBlockingQueue(1) };
+
+    @GET
+    @Path("suspend")
+    public void suspend(@Suspended AsyncResponse asyncResponse) {
+        stage[0].add(asyncResponse);
+
+        ///////////////////////////////
+//	   asyncResponse.register(SettingCompletionCallback.class).isEmpty();
```

Comment:
Could probably remove these comments... not critical.

Reply:
I tried cleaning up as many stray comments as I could, but with 500+ files I was bound to miss a few. Good catch!